src/request/comment.rs

```diff
@@ -220,4 +288,17 @@ mod tests {
             "gameVersion=21&binaryVersion=33&secret=Wmfd2893gb7&total=0&page=2&accountID=1710032"
         );
     }
+
+    #[test]
+    fn serialize_comment_history() {
+        if let Err(err) = env_logger::builder().is_test(true).try_init() {
+            // nothing to make the tests fail over
+            eprintln!("Error setting up env_logger: {:?}", err)
+        }
+
+        let request = CommentHistoryRequest::new(159782398)
+            .limit(2);
+
+        assert_eq!(super::super::to_string(request), "gameVersion=21&binaryVersion=33&secret=Wmfd2893gb7&total=0&page=0&mode=0&userID=159782398&count=2")
+    }
```

Comment:
Tests go into the `tests` directory and are supposed to be roundtrip tests to ensure bijectivity of serialization and deserialization.

Reply:
I thought there were other tests here? I'll move this one though and write a more fleshed out test case.

Reply:
Fleshed out test case can be done later, as comment history relies on The LevelComment struct, it always fails the test due to the out of order indexes and the missing index 7 all together in the raw comment data. I'll rework this some other time.

Reply:
Also in some of the request modules, there are test modules within a given request module (The code linked above is an example) There are serialize test cases here, why?